### ISSUE TITLE
games-emulation/emulationstation doesn't work.

### ISSUE THREAD

**Issue by crocket** [stefan-gr/abendbrot#102]
games-emulation/emulationstation-2.6.0 freezes when I try to enter a game system like dreamcast.
I tried to compile -9999, but it fails, too.

```
 * Messages for package games-emulation/emulationstation-9999-r2:

 * Failed Patch: emulationstation-9999-include-fix.patch !
 *  ( /var/tmp/portage/games-emulation/emulationstation-9999-r2/files/emulationstation-9999-include-fix.patch )
```

**Comment by crocket** [comment]
After upgrading emulationstation from 2.6.0 to 2.7.1, it works without freezing.

**Comment by crocket** [comment]
Unfortunately, -9999 can be compiled but cannot be executed.

```
$ emulationstation
lvl0:   Error - File type unknown!
Segmentation fault
```

**Comment by stefan-gr** [comment]
Have you tried deleting the gamelist/config and trying again? I have heard thearchive scraper can cause problems like this.

**Comment by crocket** [comment]
I moved `~/.emulationstation` somewhere else and executed `emulationstation`. I still see the same error.

**Comment by stefan-gr** [comment]
Yeah, I have the same problem on arch. But getting usable backtraces on arch is impossible.

**Comment by barbudreadmon** [comment]
I'm trying to fix -9999 version, see https://github.com/stefan-gr/abendbrot/pull/119 .

**Comment by barbudreadmon** [comment]
Actually, my ebuild is working, i was having issues because of some EGIT_COMMIT value in /etc/portage/env/games-emulation/emulationstation. Now i have a working emulationstation-2.7.5 running on kms on my raspberry.
